aten/src/ATen/native/cuda/IGammaBackwardKernel.cu

```diff
@@ -0,0 +1,32 @@
+#include <ATen/Dispatch.h>
```

Comment:
Can you add this
```suggestion
#define TORCH_ASSERT_NO_OPERATORS
#include <ATen/Dispatch.h>
```

It makes sure we don't accidentally include files that get would cause this to get rebuilt often.